content/zh/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,345 @@
+        <ul>
+          <li><code>NT AUTHORITY\SYSTEM</code></li>
+          <li><code>NT AUTHORITY\Local service</code></li>
+          <li><code>NT AUTHORITY\NetworkService</code></li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td style="white-space: nowrap"><a href="/zh/docs/concepts/security/pod-security-standards"><tt>runAsNonRoot</tt></a></td>
+      <td>
+        <p><!--Because HostProcess containers have privileged access to the host, the <tt>runAsNonRoot</tt> field cannot be set to true.-->
+        因为 HostProcess 容器有访问主机的特权，<tt>runAsNonRoot</tt> 字段不可以设置为 true。
+        </p>
+        <p><strong><!--Allowed Values-->可选值</strong></p>
+        <ul>
+          <li><!--Undefined/Nil-->未定义/Nil</li>
+          <li><code>false</code></li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<!--
+### Example manifest (excerpt) {#manifest-example}
+-->
+### 配置清单示例（片段）   {#manifest-example}
+
+```yaml
+spec:
+  securityContext:
+    windowsOptions:
+      hostProcess: true
+      runAsUserName: "NT AUTHORITY\\Local service"
+  hostNetwork: true
+  containers:
+  - name: test
+    image: image1:latest
+    command:
+      - ping
+      - -t
+      - 127.0.0.1
+  nodeSelector:
+    "kubernetes.io/os": windows
+```
+
+<!--
+## Volume mounts
+
+HostProcess containers support the ability to mount volumes within the container volume space.
+Applications running inside the container can access volume mounts directly via relative or
+absolute paths. An environment variable `$CONTAINER_SANDBOX_MOUNT_POINT` is set upon container
+creation and provides the absolute host path to the container volume. Relative paths are based
+upon the `.spec.containers.volumeMounts.mountPath` configuration.
+-->
+## 卷挂载    {#volume-mounts}
+
+HostProcess 容器支持在容器卷空间中挂载卷的能力。
+在容器内运行的应用能够通过相对或者绝对路径直接访问卷挂载。
+环境变量 `$CONTAINER_SANDBOX_MOUNT_POINT` 在容器创建时被设置为指向容器卷的绝对主机路径。
+相对路径是基于 `.spec.containers.volumeMounts.mountPath` 配置来推导的。
+
+<!--
+### Example {#volume-mount-example}
+
+To access service account tokens the following path structures are supported within the container:
+-->
+### 示例    {#volume-mount-example}
+
+容器内支持通过下面的路径结构来访问服务账好令牌：
+
+`.\var\run\secrets\kubernetes.io\serviceaccount\`
+
+`$CONTAINER_SANDBOX_MOUNT_POINT\var\run\secrets\kubernetes.io\serviceaccount\`
+
+<!--
+## Resource limits
+
+Resource limits (disk, memory, cpu count) are applied to the job and are job wide.
+For example, with a limit of 10MB set, the memory allocated for any HostProcess job object
+will be capped at 10MB. This is the same behavior as other Windows container types.
+These limits would be specified the same way they are currently for whatever orchestrator
+or runtime is being used. The only difference is in the disk resource usage calculation
+used for resource tracking due to the difference in how HostProcess containers are bootstrapped.
+-->
+## 资源约束    {#resource-limits}
+
+资源约束（磁盘、内存、CPU 个数）作用到 Job 之上，并在整个 Job 上起作用。
```

Comment:
I would avoid writing Job in title case; a reader might confuse this with the Kubernetes API kind named Job.
Another option might be to add some extra wording to clarify.

Reply:
So ... the job in this context is actually a workload, not necessarily a Job resource? I may have mistaken the "job" word here.